<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Selected</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
background-color: rgb(10, 15, 62);
color:white;
font-weight:bold;
	font-size:12px;
}
QLineEdit{
	padding:5px;
	border:none;
	border-bottom:1px solid white;

}
QPushButton{

padding:5px;
border: 1px solid white
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="6" column="2">
         <widget class="QCheckBox" name="bulk_track_box">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QCheckBox" name="bulk_album_artist_box">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QCheckBox" name="bulk_album_box">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QCheckBox" name="bulk_author_box">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="bulk_title_input"/>
        </item>
        <item row="5" column="2">
         <widget class="QCheckBox" name="bulk_genre_box">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QCheckBox" name="bulk_year_box">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="bulk_artist_input"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="bulk_album_input"/>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="bulk_title_box">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="bulk_artist_box">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="bulk_album_artist_input"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Title:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="bulk_author_input"/>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="bulk_genre_input"/>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="bulk_track_input"/>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="bulk_year_input"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Artist:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Album:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Album Artist:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Author:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Genre:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Track #:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Year:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="bulk_save_input">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
